samples/meetings-context-app/nodejs/README.md

```diff
@@ -0,0 +1,103 @@
+---
+page_type: sample
+description: This sample shows the contents of meeeting tab context object in a meeting tab and meeting context details where user can see the details of participants and meeting via bot.
```

Comment:
spelling mistake "**meeeting** "

Also check the description 
meeeting **tab context object in a meeting tab and meeting context details**

Reply:
Updated.